zerver/tests/test_push_notifications.py

```diff
@@ -1788,7 +1797,7 @@ def test_get_message_payload_gcm_personal_mention(self) -> None:
 
     def test_get_message_payload_gcm_user_group_mention(self) -> None:
         self._test_get_message_payload_gcm_mentions(
-            "mentioned", "King Hamlet mentioned @mobile_team in #Verona", "mobile_team"
+            "mentioned", "King Hamlet mentioned @mobile_team in #Verona", 3, "mobile_team"
```

Comment:
Hardcoding IDs in the test database is a bad idea, since it'll break if we end up adding new user groups in `populate_db`.  We should look up the user group ID that we actually mentioned.

Reply:
Oh, I see, this user group doesn't actually exist; it's a sample.  Let me add a comment.